config/shoryuken.yml

```diff
@@ -10,5 +10,5 @@ concurrency:         <%= ENV['WORKER_COUNT'] || 25 %>
 # The delay in seconds to pause a queue when it's empty. Default 0
 delay:               <%= ENV['WORKER_PAUSE'] || 30 %>
 queues:
-  - [<%= ENV['DEFAULT_JOB_QUEUE_NAME'] || "#{env}_feeder_default" %>, 2]
-  - [<%= ENV['FIXER_CALLBACK_QUEUE_NAME'] || "#{env}_feeder_fixer_callback" %>, 1]
+  - ["<%= prefix %>_feeder_default", 2]
+  - ["<%= prefix %>_feeder_fixer_callback" %>, 1]
```

Comment:
I think you still need the `application.rb` change.  Otherwise it'll be queuing jobs to the old queues, but the workers will be polling these new ones.